cmd/nri-vsphere/nri-vsphere.go

```diff
@@ -87,6 +88,14 @@ func checkAndSanitizeConfig(config *load.Config) {
 		config.Logrus.Fatal("missing argument `pass`, please check if password has been supplied")
 	}
 
+	if config.Args.EnableVspherePerfMetrics && config.Args.PerfMetricFile ==""{
+		if runtime.GOOS == "windows" {
```

Comment:
Do you know if there is a better way to se different default values per os?
I checked the flag os in the argument definition, but it was ignored.  @ardias

Reply:
I think this is the only to check for the operating system